src/components/Card/Card.stories.tsx

```diff
@@ -59,3 +74,181 @@ export const Dragging: StoryObj<Args> = {
     elevation: 'dragging',
   },
 };
+
+export const ButtonActionCalloutCard: StoryObj<Args> = {
```

Comment:
Wait what about putting the recipes under {component}/recipes/ via `title: 'Components/Card/recipes` or do you think the badge does the job?

Reply:
could also consider always naming the stories "xx Example"

Reply:
could also consider always naming the stories "xx Example"

Reply:
These could work, actually. I wanted to lean away from using that Recipe word in EDS, so we didn't create ambiguity on what a recipe is. but it may make sense to separate out these stories from the more normal stories. 

Let's have a chat about that in our next sync :) I can create a ticket with some options to prep discussion.